test/utils.js

```diff
@@ -69,7 +70,7 @@ function findRoute(app, path) {
 }
 
 let app
-let facilityCacheServer
+let fc
```

Comment:
@Zooloo2014 Please could you make this variable more descriptive. Abbreviating here makes it harder to read the code for somebody unfamiliar.